### PR TITLE
Anzori-Ghurtchumelia_hw2_implementation

### PR DESCRIPTION
**Pull request by Ghurtchu**

### CODE REVIEW

**Comment by Ghurtchu** [comment]
corrected small mistakes and the second commit looks better I think

**Comment by Ghurtchu** [comment]
corrected small mistakes, third commit looks better

File: Java2020/src/lesson201130/ArraysClassUsage.java [hunk 1]
```diff
@@ -29,8 +29,30 @@ public static void main(String[] args) {
...
+				} 
+		// if the index is in between 1 and the length of the array, I put the key at the 'a[index]' and shift other values to the right
+		else { 
+			for(int i = 0; i < newArray.length; i++) { 
+				if(i < index) newArray[i] = a[i]; 
+				else if(i > index) newArray[i] = a[i-1]; 
+				else newArray[index] = key; 
+			}	
+		}
+		return newArray;
```
Comment (zstudent) [inline Java2020/src/lesson201130/ArraysClassUsage.java]: use arrayCopy

Comment (Ghurtchu) [reply Java2020/src/lesson201130/ArraysClassUsage.java]: noted

**Pull request closed without merging**